File: C_Programming/Assignment_5/prime.c
#include<stdio.h>
int prime(int);
void main(){
    int n;
    printf("enter the number\n",n);
    scanf("%d",&n);
    int x=prime(n);
    if(x==1)
    printf("the number is prime");
    else
    printf("not prime");
}
int prime(int n)
{
    int i=2,flag=0;
    while(i<n)
    {
       if(n%i==0){
        flag=1;
        break;
       }
       i++;
    }
       if(flag==0)
       return 1;
       else
       return 0;
 }

File: Java Programms/Array/src/p1/Prime.java
package p1;

public class Prime {

	public static void main(String[] args) {
		int[] arr = {10, 33, 11, 7, 13};
		int size = arr.length;

		System.out.println("Prime numbers in array:");

		for (int i = 0; i < size; i++) {
			int flag = 0;

				for (int j = 2; j <= arr[i] / 2; j++) {
					if (arr[i] % j == 0) {
						flag = 1;
						break;
					}
				}
			

			if (flag == 0) {
				System.out.println(arr[i]);
			}
		}
	}
}
